Lib9c/Action/ClaimStakeReward.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Immutable;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    public class ClaimStakeReward : GameAction
+    {
+        internal Address AvatarAddress { get; private set; }
+
+        public ClaimStakeReward(Address avatarAddress)
+        {
+            AvatarAddress = avatarAddress;
+        }
+
+        public ClaimStakeReward() : base()
+        {
+        }
+
+        public override IAccountStateDelta Execute(IActionContext context)
+        {
+            var states = context.PreviousStates;
+            if (!states.TryGetStakeState(context.Signer, out StakeState stakeState))
+            {
+                throw new InvalidOperationException();
```

Comment:
we already implemented `FailedLoadStateException` this case.

Reply:
I applied it with [3f73165](https://github.com/planetarium/lib9c/pull/962/commits/3f7316537df86363ac79d1ffc901776097dfb5c8)